peer/peer.go

```diff
@@ -811,39 +811,38 @@ func (p *Peer) pushVersionMsg() error {
 }
 
 // PushAddrMsg sends an addr message to the connected peer using the provided
-// addresses.  This function is useful over manually sending the message via
+// addresses. This function is useful over manually sending the message via
 // QueueMessage since it automatically limits the addresses to the maximum
 // number allowed by the message and randomizes the chosen addresses when there
-// are too many.  No message will be sent if there are no entries in the
-// provided addresses slice.
-// It is safe for concurrent access.
-func (p *Peer) PushAddrMsg(addresses []*wire.NetAddress) ([]*wire.NetAddress, error) {
+// are too many. It returns the addresses that were actually sent and no message
+// will be sent if there are no entries in the provided addresses slice.
+//
+// This function is safe for concurrent access.
+func (p *Peer) PushAddrMsg(addresses []*wire.NetAddress) (
+	[]*wire.NetAddress, error) {
```

Comment:
Please leave this on the same line.  Almost everywhere else in the project function definitions don't break across lines unless they are excessively long.  Aside from just general consistency which is probably the most important reason, we've also received requests to do it because splitting it doesn't work well with the intellisense features of some editors or godoc.